snap/snapcraft.yaml

```diff
@@ -0,0 +1,78 @@
+name: lapce
+adopt-info: lapce
+summary: Lightning-fast And Powerful Code Editor
+description: |
+    Lapce (IPA: /læps/) is written in pure Rust with a UI in Floem. It is designed with Rope Science
+    from the Xi-Editor which makes for lightning-fast computation, and leverages Wgpu for rendering. 
+    More information about the features of Lapce can be found on the main website and user
+    documentation can be found on GitBook.
+
+base: core20
```

Comment:
Would the newer `core22` base work? I'd rather keep all components on latest versions

Reply:
sure that's easy enough to change.